src/controllers/lyrics.html

```diff
@@ -0,0 +1,4 @@
+<div id="lyricPage" data-role="page" class="page lyricPage" data-backbutton="true">
+    <div class="dynamicLyricsContainer padded-bottom-page">
+    </div>
```

Comment:
For some reason `flex` inside another `flex` has the wrong height - the now playing bar overlaps the last line.
Placing it in another `div` fixes the problem.
```suggestion
    <div>
        <div class="dynamicLyricsContainer padded-bottom-page">
        </div>
    </div>
```